front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -255,3 +267,10 @@ AgentTablesQueryAction.belongsTo(AgentMessage, {
 AgentMessage.hasMany(AgentTablesQueryAction, {
   foreignKey: { name: "agentMessageId", allowNull: false },
 });
+
+FileModel.hasMany(AgentTablesQueryAction, {
+  foreignKey: { name: "resultsFileId", allowNull: true },
```

Comment:
Let's be explicit I don't think we want any cascade operation here, do we?

Reply:
default is set null here

Reply:
(Which is what I want)

Reply:
Can we then be explicit about it? Cause I recall the default value to be "ON DELETE CASCADE".

Reply:
It's cascade when the FK is non-nullable. SET NULL when it's nullable

Reply:
Ok then let's be explicit about this Sequelize wizardry.

Reply:
If really it makes you happier I'll do it :)

Reply:
It does 🤗 !